TinCan/ActivityDefinition.cs

```diff
@@ -13,7 +13,16 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 */
+
+/*
+ *  Modified by Paul Carpenter 2015
+ *  
+ *  Includes support for Activity Interactions
+ *  
+ */
```

Comment:
Would prefer to keep this out of the source, git should be able to track these types of changes easily. If we need new attribution it can go somewhere outside of the source itself.